settings.gradle.kts

```diff
@@ -1,8 +1,35 @@
 plugins {
     id("com.gradle.develocity") version "3.17.6"
-    id("com.gradle.enterprise.gradle-enterprise-conventions-plugin") version "0.7.5"
+    id("com.gradle.common-custom-user-data-gradle-plugin") version "2.0.2"
+}
+
+val isCI = providers.environmentVariable("CI").isPresent
+
+develocity {
+    server = "https://ge.gradle.org"
+    buildScan {
+        uploadInBackground = !isCI
+        publishing.onlyIf { it.isAuthenticated }
+        obfuscation {
+            ipAddresses { addresses -> addresses.map { "0.0.0.0" } }
+        }
+    }
+}
+
+buildCache {
+    local {
+        isEnabled = true
+    }
+
+    remote(develocity.buildCache) {
+        server = "https://eu-build-cache.gradle.org"
```

Comment:
same comment as the other PR, why are we pushing to this node? how is replication set up between all the nodes? also which node is closest to CI?

Reply:
See [thread](https://gradle.slack.com/archives/C032LFSKY/p1722411224756959). The default for ge.gradle.org is not used and eu node was the default till now, so that was the decision (to leave it as default).